Index: Scripts/dul-receive-pack
===================================================================
diff --git a/Scripts/dul-receive-pack b/Scripts/dul-receive-pack
--- a/Scripts/dul-receive-pack	
+++ b/Scripts/dul-receive-pack	
@@ -19,10 +19,11 @@
 # License, Version 2.0.
 #
 
-from dulwich.porcelain import receive_pack
 import os
 import sys
 
+from dulwich.porcelain import receive_pack
+
 if len(sys.argv) < 2:
     sys.stderr.write("usage: %s <git-dir>\n" % os.path.basename(sys.argv[0]))
     sys.exit(1)
Index: Scripts/dul-upload-pack
===================================================================
diff --git a/Scripts/dul-upload-pack b/Scripts/dul-upload-pack
--- a/Scripts/dul-upload-pack	
+++ b/Scripts/dul-upload-pack	
@@ -19,10 +19,11 @@
 # License, Version 2.0.
 #
 
-from dulwich.porcelain import upload_pack
 import os
 import sys
 
+from dulwich.porcelain import upload_pack
+
 if len(sys.argv) < 2:
     sys.stderr.write("usage: %s <git-dir>\n" % os.path.basename(sys.argv[0]))
     sys.exit(1)
